src/main/java/fr/icefeather/wsn/client/application/App.java

```diff
@@ -66,14 +67,17 @@ public App() {
 
         final AppForm formulaire = new AppForm();
 
-        final AppTextField urlTextField = new AppTextField(TextUrl, "http:// url broker wsn", "http://localhost/cxf/wsn/NotificationBroker", true);
+        final AppTextField urlTextField = new AppTextField(TextUrl, "http:// url broker wsn", "http://localhost:8181/cxf/wsn/NotificationBroker", true);
```

Comment:
**REJECT** : Don't force port

Reply:
le fait de forcer le port dans l'url par défaut n'a pas d'intérêt (dans une version globale tout du moins)

Reply:
in the local install we have the cxf installed on the port 8181, its easier for localhost tests